### PR TITLE
Fix/small changes

### LINKED ISSUE #398: Add Strategy name in the backtesting overview
Right now there is no confirmation in the backtest overview of what strategy has been run. For clarity, this should be added in the 'settings' section of the backtesting report.

### PR DESCRIPTION
**Pull request by WillemGerritzen**
# Results of merging this

Fixes #398 

# What has been changed?

* Changed 'Winning weeks' column and row to 'Weekly market perf. (W/D/L)'
* Reduced the testing range and timeframe in the default config file to 30m and 6 months
* Add the name of the strategy in the settings table
* Changed the title of the backtesting overview
* Changed the row label 'BTC' to 'BTC/USDT'
* Add an extra warning for out-of-date engines at the start of the script

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   "exchange": "binance",
-  "timeframe": "5m",
+  "timeframe": "30m",
   "max-open-trades": 3,
   "exposure-per-trade": 100,
   "starting-capital": 1000,
-  "backtesting-from": "2020-09-01",
-  "backtesting-to": "2021-09-01",
+  "backtesting-from": "2021-05-01",
+  "backtesting-to": "2021-11-01",
```
Comment (sheridavandenbent) [inline config.json:8]:
```suggestion
  "backtesting-from": "2021-01-01",
  "backtesting-to": "2021-07-01",
```

Maybe have it from the first of January? That looks a bit neater!

File: modules/setup/config/__init__.py [hunk 2]
```diff
@@ -20,11 +20,14 @@
 hour = 60 * minute
 day = 24 * hour
 
+STRATEGY_NAME: str
+
```
Comment (sheridavandenbent) [inline modules/setup/config/__init__.py:24]: What is this constant used for?

File: modules/setup/config/__init__.py [hunk 3]
```diff
@@ -58,6 +61,7 @@ async def create(args):
         exchange_str = config["exchange"]
         config_module.timeframe = config["timeframe"]
         config_module.timeframe_ms = parse_timeframe(config_module.timeframe)
+        ConfigModule.strategy_name = config['strategy-name']
```
Comment (sheridavandenbent) [inline modules/setup/config/__init__.py:64]:
```suggestion
        config_module.strategy_name = config['strategy-name']
```

Shouldn't it modify the instance of the class, not the main class itself?

**Review by sheridavandenbent — CHANGES_REQUESTED** [review]
See comments!

File: config.json (hunk 1 above)
Comment (WillemGerritzen) [reply config.json:8]: Indeed!

File: modules/setup/config/__init__.py (hunk 3 above)
Comment (WillemGerritzen) [reply modules/setup/config/__init__.py]: Changes have been made.

**Review by sheridavandenbent — APPROVED** [review]

**Pull request merged**